### PR TITLE
Improve logs for transfer task validator

### PR DESCRIPTION
**Pull request by Shaddoll**
**What changed?**
- Improve logs for transfer task validator to capture a potential false positive case

**Why?**
- To improve transfer task validator

**How did you test it?**

**Potential risks**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: service/history/queue/transfer_queue_validator.go [hunk 1]
```diff
@@ -42,9 +42,9 @@ const (
 
 type (
 	pendingTaskInfo struct {
-		executionInfo    *persistence.WorkflowExecutionInfo
-		task             persistence.Task
-		persistenceError bool
+		executionInfo          *persistence.WorkflowExecutionInfo
+		task                   persistence.Task
+		potentialFalsePositive bool
```
Comment (davidporter-id-au) [inline service/history/queue/transfer_queue_validator.go:47]: what will be the action upon this flag?

Comment (davidporter-id-au) [reply service/history/queue/transfer_queue_validator.go:47]: Should this log or something instead?

Comment (Shaddoll) [reply service/history/queue/transfer_queue_validator.go:47]: it's a tag in the logs

Comment (davidporter-id-au) [reply service/history/queue/transfer_queue_validator.go:47]: ah! I didn't see, thank you

**Review by davidporter-id-au — APPROVED** [review]

**Pull request merged**